src/content.js

```diff
@@ -53,8 +53,8 @@ function groupByRepo(events) {
 		return repos;
 	}, new Map());
 }
-function apply(options, insertionPoint) {
-	const holder = domifyEscape`<div class="ghcf-holder"><i>`;
+function apply(options) {
+	const holder = document.createDocumentFragment();
```

Comment:
I see what the reason for this is: avoiding `Page X`. I want to keep that for now and solve #7 instead.

Also I remember that I used `<i>` to trick GitHub's own CSS to add a line above the first element:

<img width="128" alt="screen shot 2017-06-07 at 19 45 28" src="https://user-images.githubusercontent.com/1402241/26876711-03afd2c2-4bba-11e7-8985-e1855bfd83df.png">